cord19-explorer/src/main/webapp/WEB-INF/views/layouts/footer.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+</head>
+<body>
+	<div th:fragment="footer" th:align="center">Cord19 Explorer version 2.0</div>
```

Comment:
this needs to pull the version from the pom (similar to other apps like conceptpower).